pyspark/bigdl/util/common.py

```diff
@@ -247,6 +247,22 @@ def uniform(self, a, b, size):
 
 def init_engine(bigdl_type="float"):
     callBigDlFunc(bigdl_type, "initEngine")
+    print('Setting current log level to "INFO".')
+    print("To adjust logging level use sc.setLogLevel(newLevel).")
+    sc = get_spark_context()
+    sc.setLogLevel("INFO")
+
+
+def redireSparkInfoLogs(disableRedirect="false", enableSparkLogRedirect="true",
+                        logPath=os.getcwd()+"/bigdl.log", bigdl_type="float"):
+    """
+    Redirect Spark INFO level logs to the specified path.
```

Comment:
What's the relationship between `enableSparkLogRedirect` and `disableRedirect`?
Should we just simply use "enableSparkLogRedirect" and "enableBigDLLogRedirect" ?
Only redirect the INFO level? how about those troubling WARNING we've seen before?

Reply:
1. I think 'enableSparkLogRedirect' is to enable redirecting log from spark. 'disableRedirect' is to disable redirecting log from both spark and bigdl.
2. These two parameters are in the original code in Scala and I don't change them. If you want to make the change, maybe we should discuss with Yanzhang as this will affect the parameters in the Scala side.